.github/workflows/flatpak.yml

```diff
@@ -40,13 +40,14 @@ jobs:
         id: setup
         run: |
           echo "::set-output name=commitHash::$(git rev-parse --short=9 HEAD)"
+          echo "::set-output name=manifestHash::$(sha256sum CI/flatpak/com.obsproject.Studio.json | cut -f1 -d' ')"
 
       - name: Build Flatpak Manifest
         uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@master
         with:
           bundle: obs-studio-${{ steps.setup.outputs.commitHash }}.flatpak
           manifest-path: CI/flatpak/com.obsproject.Studio.json
-          cache-key: flatpak-builder-${{ github.sha }}
+          cache-key: flatpak-builder-${{ steps.setup.outputs.manifestHash }}
```

Comment:
This won't work really, the issue comes from https://github.com/actions/toolkit/issues/658 and once fixed the cache-key should no longer be needed to be set by OBS or any of the action users.

Using the current commit is less error prone than the manifest one from experience

Reply:
It seems to me that this comment is recommending that we continue using `github.sha`.